Update conda skeleton documentation to add link to Grayskull (#5027)
conda/conda-build: docs/source/user-guide/tutorials/build-pkgs-skeleton.rst

   :local:
   :depth: 1

> .. tip::

Reviewer: `.. admonition::` doesn't work does it

> .. warning::

Author: ```suggestion
.. tip::
```

>    For a better experience with building conda packages from PyPI pacakages, we recommend the usage

Reviewer: ```suggestion
   For a better experience with building conda packages from PyPI packages, we recommend the usage
```

Reviewer: I would say "We recommend Grayskull, a newer alternative to conda-skeleton, to generate conda recipes for PyPI packages."

>    of `Grayskull <https://github.com/conda/grayskull>`. Please check out their project page on GitHub

Author: ```suggestion
   of `Grayskull <https://github.com/conda/grayskull>`_. Please check out their project page on GitHub
```

>    For a better experience with building conda packages from PyPI pacakages, we recommend the usage
>    of `Grayskull <https://github.com/conda/grayskull>`_. Please check out their project page on GitHub

Author: ```suggestion
   We recommend `Grayskull <https://github.com/conda/grayskull>`_, a newer alternative to conda-skeleton,
   to generate conda recipes for PyPI packages. Please check out their project page on GitHub
```
